AudioLinkUnityProject/Packages/com.vrchat.core.vpm-resolver/package.json

```diff
@@ -1,17 +1,17 @@
 {
-  "name": "com.vrchat.core.vpm-resolver",
-  "displayName": "VRChat Package Resolver Tool",
-  "version": "0.1.15",
-  "unity": "2019.4",
-  "description": "Tool to Download VPM Packages",
-  "vrchatVersion": "2022.1.1",
-  "author": {
-    "name": "VRChat",
-    "email": "developer@vrchat.com",
-    "url": "https://github.com/vrchat/packages"
-  },
-  "url": "",
-  "dependencies": {
-    "com.unity.nuget.newtonsoft-json": "2.0.2"
-  }
-}
\ No newline at end of file
+	"name" : "com.vrchat.core.vpm-resolver",
+	"displayName" : "VRChat Package Resolver Tool",
+	"version" : "0.1.17",
+	"unity" : "2019.4",
+	"description" : "Tool to Download VPM Packages",
+	"vrchatVersion" : "2022.1.1",
+	"author" : {
+		"name" : "VRChat",
+		"email" : "developer@vrchat.com",
+		"url" : "https://github.com/vrchat/packages"
+	},
+	"url" : "",
+	"dependencies" : {
+		"com.unity.nuget.newtonsoft-json" : "2.0.2"
+	}
+}
```

Comment:
I mean, this is fine.. but please with the unrelated diffs in your PR's x)

Reply:
sorry, but the manual updating was really annoying me